addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxServerHandler.java

```diff
@@ -556,24 +560,66 @@ public void updateListener(SqueezeBoxPlayerEventListener listener) {
             }
         }
 
+        /**
+         * Returns a hexadecimal encoded MD5 hash for the input String.
+         *
+         * @param data
+         * @return
+         */
+        private String getMD5Hash(String data) {
+            String result = null;
+            try {
+                MessageDigest digest = MessageDigest.getInstance("MD5");
+                byte[] hash = digest.digest(data.getBytes("UTF-8"));
+                return bytesToHex(hash); // make it printable
+            } catch (Exception ex) {
+                ex.printStackTrace();
```

Comment:
You should not use `printStackTrace()` instead pass the exception as the last argument of a logging statement that way it ends up where the user would expect it and can be potentially suppressed if needed.